front/app/hooks/useProjectGroups.ts

```diff
@@ -2,23 +2,23 @@ import { useState, useEffect } from 'react';
 import { Observable, of } from 'rxjs';
 import { map } from 'rxjs/operators';
 import {
-  IGroupsProjectsData,
-  groupsProjectsByProjectIdStream,
-} from 'services/groupsProjects';
+  IProjectGroupData,
```

Comment:
You can also add a quick test for the hook. We have a standard way of doing this described in our [testing doc](https://www.notion.so/citizenlab/Testing-253d0c3cd99841a59929f7f615179935).